widget_driver_generator/test/utils/string_extensions_test.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:test/test.dart';
+import 'package:widget_driver_generator/src/utils/string_extensions.dart';
+
+void main() {
+  group('StringExtensions:', () {
+    group('removeLeadingUnderScore:', () {
+      test('should return as is, if no String is empty', () {
+        const testString = '';
+        expect(testString.removeLeadingUnderscore(), testString);
+      });
+      test('should return as is, if first character is no underscore', () {
```

Comment:
Same here. I think it becomes a bit more readable and clear that these are different tests if you add a newline..

```
      test('should return as is, if no String is empty', () {
        const testString = '';
        expect(testString.removeLeadingUnderscore(), testString);
      });
      test('should return as is, if first character is no underscore', () {
        const testString = 'hallo';
        expect(testString.removeLeadingUnderscore(), testString);
      });
      test('should remove leading underscore if it exists', () {
        const testString = '_hallo';
        expect(testString.removeLeadingUnderscore(), 'hallo');
      });
```
or 
```
      test('should return as is, if no String is empty', () {
        const testString = '';
        expect(testString.removeLeadingUnderscore(), testString);
      });

      test('should return as is, if first character is no underscore', () {
        const testString = 'hallo';
        expect(testString.removeLeadingUnderscore(), testString);
      });

      test('should remove leading underscore if it exists', () {
        const testString = '_hallo';
        expect(testString.removeLeadingUnderscore(), 'hallo');
      });
```

Tests are not only meant to be executed by the computer.. 
They are also meant to be read by other developers.. 
I think you get a bit better overview when quickly looking over this code with the newlines